features/db-size.feature

```diff
@@ -133,6 +133,15 @@ Feature: Display database size
     When I run `wp db size --size_format=TB`
     Then STDOUT should be a number
 
+  Scenario: Display only database size in megabytes with sprcic precision for a WordPress install
+    Given a WP install
+
+    When I run `wp db size --size_format=mb --decimals=1`
+    Then STDOUT should be:
+      """
+      5.9
+      """
```

Comment:
You cannot use an actual size here, as that will fluctuate across DB implementations and make for a flaky test.
So we're just looking for the period instead.
```suggestion
	When I run `wp db size --size_format=mb --decimals=0`
    Then STDOUT should not contain:
      """
      .
      """

    And STDOUT should not contain:
      """
      MB
      """

	When I run `wp db size --size_format=mb --decimals=1`
    Then STDOUT should contain:
      """
      .
      """

    And STDOUT should not contain:
      """
      MB
      """
```